packages/react-ux-capture-example/src/TimeLabel.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const TimeLabel = props => {
+	const { time, label, emoji } = props;
```

Comment:
Since 'emoji' can be anything, might be better to call it 'icon' or something.

Also, [use composition](https://jxnblk.com/blog/defining-component-apis-in-react/#use-composition) and [keep props APIs parallel](https://jxnblk.com/blog/defining-component-apis-in-react/#keep-props-apis-parallel) - the `time` prop could be more elegantly expressed as `children`, and the `label` prop is just the `title` attribute, so a more predictable/conventional interface would look like

```jsx
<TimeLabel title="foo" icon="🕓">250ms</TimeLabel>
```

I also recommend not hardcoding the units, unless you rename this to `TimeLabelMillis` or something that indicates the implicit value constraint.

If you wanted to get really semantic, a [`<time>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) element would be most 'correct' here.

Reply:
Milliseconds are the unit #webperf works on, seconds are way too optimistic and nano-seconds are unrealistic. So I would keep it there for now or make it a prop with a default.

I'll see if <time> makes sense, but it seems to deal with higher numbers generally. Will see if it works here.